pyaedt/application/Analysis.py

```diff
@@ -1849,3 +1850,104 @@ def value_with_units(self, value, units=None):
                 units = self.modeler.model_units
             val = "{0}{1}".format(value, units)
         return val
+
+    # Add test and check if Analysis.py is the best place to have the method
+    @pyaedt_function_handler()
+    def export_rl_matrix(
+        self,
+        matrix_name,
+        file_path,
+        is_format_default=True,
+        width=8,
+        precision=2,
+        is_exponential=False,
+        setup_name=None,
+        default_adaptive=None,
+        is_post_processed=False,
+    ):
+        """Export R/L matrix after solving.
+
+        Parameters
+        ----------
+        matrix_name : str
+            Matrix name to be exported.
+        file_path : str
+            File path to export R/L matrix file.
+        is_format_default : bool, optional
+            Whether the exported format is default or not.
+            If False the custom format is set (no exponential).
+        width : int, optional
+            Column width in exported .txt file.
+        precision : int, optional
+            Decimal precision number in exported .txt file.
+        is_exponential : bool, optional
+            Whether the format number is exponential or not.
+        setup_name : str, optional
+            Name of the setup.
+        default_adaptive : str, optional
+            Adaptive type.
+        is_post_processed : bool, optional
+            Boolean to check if it is post processed.
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+        """
+        try:
+            if not self.solution_type == "EddyCurrent":
+                msg = "RL Matrix can only be exported if solution type is Eddy Current."
+                raise ValueError(msg)
```

Comment:
Usually, it is not needed to create a new variable when it is used only one time.
```suggestion
                raise ValueError("RL Matrix can only be exported if solution type is Eddy Current.")
```